src/Controller.h

```diff
@@ -69,17 +69,16 @@ class Controller
   void setFrameHandler(void (*fptr)(CANFrame *msg), byte *opcodes = NULL, byte num_opcodes = 0);
 
 private:                                          // protected members become private in derived classes
-  CANFrame _msg;
   UserInterface *_ui;
   Configuration *module_config;
   std::initializer_list<Service *> services;
   Transport * transport;
 
   unsigned char *_mparams;
   unsigned char *_mname;
-  void (*framehandler)(CANFrame *msg);
-  byte *_opcodes;
-  byte _num_opcodes;
+  void (*framehandler)(CANFrame *msg) = NULL;
+  byte *_opcodes = NULL;
+  byte _num_opcodes = 0;
```

Comment:
While working on test cases I noticed a few cases where these fields were not null/zero which caused problems. 
All other fields are set when the controller is created.